### ISSUE TITLE
The code to output the mask?

### ISSUE THREAD

**Issue by VCBE123** [wulalago/UNet#6]
Now I have a trained model,  how to get the segmentation results for visualization?

**Comment by wulalago** [comment]
I didn't upload the inference code, but you can use the code as follow which I will added to the repo this month:

`net.load_state_dict(torch.load(your_model_path))`
`net.eval()`
`for image in loader:  # the loader is your test loader`
`    predict = net(image.cuda())`
`    predict = predict.cpu().data.numpy()`
`    predict_nii = nibabel.Nifti1Image(predict, affine=None)`
`    nibabel.save(predict_nii, 'your_save_path')`

After saving nifti image of predict, you can visualize it by itk-snap
